examples/react-urql-retry/src/pages/Color.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { gql, useQuery } from "urql";
+
+const RANDOM_COLOR_QUERY = gql`
+  query RandomColor {
+    randomColor {
+      name
+      hex
+    }
+  }
+`;
+
+const RandomColorDisplay = () => {
+  const [result] = useQuery({ query: RANDOM_COLOR_QUERY });
+
+  const { data, fetching, error } = result;
+
+  return (
+    <div>
+      {fetching && <p>Loading...</p>}
+
+      {error && <p>Oh no... {error.message}</p>}
+
+      {data && (
+        <div style={{ backgroundColor: data.randomColor.hex }}>
+          {data.randomColor.name}
+        </div>
+      )}
```

Comment:
One nice thing you could add to differentiate it from the other example is something like:

```suggestion
      )}
      
      {result.operation && (
        <p>We retried {res.operation.context.retryCount} times to get a result without an error.</p>
      )}
```